data/CO/incentives.json

```diff
@@ -8909,21 +8948,21 @@
       "homeowner"
     ],
     "short_description": {
-      "en": "$100 rebate for electric-to-induction cooktops/ranges (measuring 30\" or larger)\n$350 rebate for gas-to-induction or new construction cooktops/ranges"
+      "en": "$100 rebate for electric-to-induction cooktops/ranges (measuring 30\" or larger)$350 rebate for gas-to-induction or new construction cooktops/ranges."
```

Comment:
Ugh, in this one the newline should have been replaced by punctuation and not just a space :(